ChangeLog

```diff
@@ -7,6 +7,16 @@ What's New in Pylint 2.2?
 
 Release date: TBA
 
+   * PyLint comments now officially work with other linter comments
+   
```

Comment:
Are you sure your PR actually fixes all these bugs? From what I'm noticing:
- #2297 is already fixed in master
- #2470 is also fixed, and I cannot reproduce with your given example
- as well as #2485, which again I cannot reproduce using your example from https://github.com/PyCQA/pylint/issues/2485#issuecomment-427626955.

Unless you are testing with a completely different code than the one posted in the aforementioned comment, I don't think your PR "fixes" these issues, as they are already fixed. What instead is fixing is that it correctly supports long lines and additional disables, but that's a separate bug.